for-developers/hats-protocol-overview/renouncing-hats.md

```diff
@@ -1,3 +1,22 @@
 # Renouncing Hats
 
-The wearer of a hat can "take off" their hat via `Hats.renounceHat`. This burns the token and revokes any associated authorities and responsibilities from the now-former wearer, but does not put the wearer in bad standing.
+The wearer of a hat can "take off" their hat via `Hats.renounceHat`.
+This burns the token and revokes any associated authorities and responsibilities from the now-former wearer, but does not put the wearer in bad standing.
+
+## How do I renounce a Hat?
+
+1. **Sign in with the correct wallet.**
+   The address you sign in with must be a wearer of the hat you intend to renounce.
+
+2. **Visit the Hats Protocol App:** https://app.hatsprotocol.xyz/
+
+3. **Select the hat you wish to Renounce.**
+   You will now see a Hat's tree at a url like: `https://app.hatsprotocol.xyz/trees/...`
+
+4. **Click Renounce in the corresponding kebab-menu.**
+   In the right side-panel, you will see a heading named "Hat Wearers". Click the kebab-menu (the "more" button), then click renounce.
+
+**Note:** You will only see the option to renounce your Hats, you will not see renounce as an option for other wearers of the same Hat.
+
+## What's next?
+Remove Signing Authority via the HSG Portal. This maintenance step can be performed by any wallet to sync the multisig safe. 
```

Comment:
This is specific to HatsSignerGate and not necessarily relevant to Hats Protocol users in general. If you think it should be included, please add it to this page: https://github.com/Hats-Protocol/docs/blob/main/hats-integrations/hat-gated-authorities/safe-multisig-signing-authority.md